task/09-functions-n-closures-tasks.js

```diff
@@ -104,10 +99,17 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+  return () => {
+    for (let i = 0; i <= attempts; i++) {
```

Comment:
you have 2 exit conditions for the for-loop. In this case they are the same.  Keep only one of them

![image](https://user-images.githubusercontent.com/7272208/87838898-343e2800-c8a1-11ea-9ad0-5be14dc33dd5.png)